doc/source/getting_started/code/ODL Basics.py

```diff
@@ -0,0 +1,198 @@
+
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# # ODL Basics
+
+# Welcome to your first tutorial. Here you will learn the basics of linear
+# spaces, vectors and related functionality as used throughout ODL. The
+# main purpose is to get you accustomed to working with the library, the
+# "look and feel" of typical code and some building blocks and patterns
+# that repeat themselves in other parts. We will explain them as we go
+# along.
+#
+# At this point, you should have a working installation of ODL on your
+# machine. If that's not the case, check out [the installation instruction
+# s](https://odlgroup.github.io/odl/getting_started/installing.html) and
+# come back when you're finished!
+
+# We import this for Python 2 compatibility
+from __future__ import print_function
```

Comment:
no real need, just put parenthesis around and you're fine (unless you use some arguments to the function or use print as a first class object, which I guess you dont).